pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -35,7 +37,7 @@ mkYarnPackage rec {
   packageJSON = ./element-desktop-package.json;
   yarnNix = ./element-desktop-yarndeps.nix;
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper makeicns ];
```

Comment:
That is missing a conditional on darwin. Should fix the eval error.

Reply:
Whoops, missed the conditional.  Should be fixed now.